tests/test_route.py

```diff
@@ -0,0 +1,48 @@
+from urllib.request import urlopen
+
+import pytest
+from flask import url_for
+
+
+@pytest.mark.usefixtures("live_server")
+class TestNotificationEndpoint:
+    def test_notification_route_response(self):
+        """
+        GIVEN: function runs to connect with send_notification route/endpoint.
+        WHEN: function successfully connects with the endpoint.
+        THEN: function check the response 200(successful) in return.
+       """
```

Comment:
:pick: Not a massive problem but...try to use the docstrings here to describe the behaviour of the test rather than a description - the docstrings should add to the code not merely describe it. 

**Exemplar:** 
GIVEN our service running on a flask test client
WHEN we make a GET request to the send notification endpoint
THEN we expect a successful 200 back.

Reply:
updated 👍